### PR TITLE
DT-2757 Sentence Summary Changes

### PR DESCRIPTION
**Pull request by Mjwillis**
- Add fine to sentence since this is used in DPS
- Remove CRD override that and calculate CRD correctly
- Remove effective sentence length since we don’t know how reliable it is

### CODE REVIEW

**Review by andymarke — DISMISSED** [review]
LGTM

File: src/test/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/OffenderPendingDeletionRepositoryTest.java [hunk 1]
```diff
@@ -32,30 +30,6 @@
     @Autowired
     private OffenderPendingDeletionRepository repository;
 
-    @Test
```
Comment (Mjwillis) [inline src/test/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/OffenderPendingDeletionRepositoryTest.java:35]: I've removed these tests as this code will be deleted once the new delete data from nomis service is complete from @hyderkhanmoj

Comment (hyderkhanmoj) [reply src/test/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/OffenderPendingDeletionRepositoryTest.java:35]: Good to remove as it's replaced in this PR: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerdeletionservice/repository/jpa/DeceasedOffenderPendingDeletionRepositoryTest.kt

**Review by andymarke — APPROVED** [review]
Lgtm

File: src/test/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/OffenderPendingDeletionRepositoryTest.java (hunk 1 above)
Comment (Mjwillis) [reply src/test/java/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/OffenderPendingDeletionRepositoryTest.java:35]: @hyderkhanmoj I've approved that PR

**Pull request merged**